src/libs/PolicyUtils.ts

```diff
@@ -8,6 +8,22 @@ type MemberEmailsToAccountIDs = Record<string, number>;
 type PersonalDetailsList = Record<string, PersonalDetails>;
 type UnitRate = {rate: number};
 
+let allPolicyTags: OnyxCollection<PolicyTags> = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.POLICY_TAGS,
+    waitForCollectionCallback: true,
+    callback: (value) => {
+        if (!value) {
+            return;
+        }
+        allPolicyTags = Object.fromEntries(Object.entries(value).filter(([, policyTags]) => !!policyTags));
+    },
+});
+
+function getPolicyTags(policyID: string) {
```

Comment:
Exporting methods like this (which only return something from an Onyx collection) are a bad pattern. I am trying to get rid of them in https://github.com/Expensify/App/issues/27262. All that needs to happen here is move the Onyx connection to the `ModifiedExpenseMessage` file.

Reply:
fixed.